UM/PluginRegistry.py

```diff
@@ -515,8 +514,9 @@ def uninstallPlugin(self, plugin_id: str) -> Dict[str, str]:
 
             # Remove the plugin object from the Plugin Registry:
             self._plugins.pop(plugin_id, None)
-            self._plugins_installed.remove(plugin_id)
-
+            if plugin_id in self._plugins_installed:
+                self._plugins_installed.remove(plugin_id)
```

Comment:
this whole code block executes `if not in_to_install`. Might I suggest removing the else statement entirely and executing it in for all cases. 

While I cannot think of any cases where a plugin would be the `_plugins_installed` object while also being in the `_plugins_to_install` object, I think it's best to cover all these edge cases. The internal check prevent plugins from being removed if they are not present; so it won't break stuff if it gets executed "incorrectly".

Reply:
https://github.com/Ultimaker/Uranium/commit/68d3c1508e23fd79dab1ae4fefcb15cce1695909